test/ibmq/test_basic_server_paths.py

```diff
@@ -36,6 +43,17 @@ def setUpClass(cls, providers):
         super().setUpClass()
         cls.providers = providers  # Dict[str, AccountProvider]
         cls.last_week = datetime.now() - timedelta(days=7)
+        # Setup list of jobs to check have been deleted.
+        cls.jobs_to_delete: List[Tuple[AccountProvider, str]] = []
+
+    def tearDown(self) -> None:
+        super().tearDown()
+        # Ensure all jobs are deleted.
+        for provider, job_id in self.jobs_to_delete:
+            try:
+                provider.runtime.delete_job(job_id)
```

Comment:
`provider.runtime.delete_job()` is only used to delete runtime jobs. These tests submit circuit jobs, not runtime jobs. `IBMQTestCase` is also supposed to automatically [record](https://github.com/Qiskit/qiskit-ibmq-provider/blob/master/test/ibmqtestcase.py#L81), [cancel](https://github.com/Qiskit/qiskit-ibmq-provider/blob/master/test/ibmqtestcase.py#L99), and [delete](https://github.com/Qiskit/qiskit-ibmq-provider/blob/master/test/ibmqtestcase.py#L101) jobs.

Reply:
Updated! Very cool, did not know `IBMQTestCase` handled all that!